README.md

```diff
@@ -1,3 +1,5 @@
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/d122f3edbb6e4ebfa8c74c4b219ce3a2)](https://www.codacy.com/app/Shamanou/FAIRifier_2?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Shamanou/FAIRifier&amp;utm_campaign=Badge_Grade)
```

Comment:
Is the codacy badge linked to Shamanou/FAIRifier_2 or to DTL-FAIRData/FAIRifier? Ideally it should be added to the DTL-FAIRData one. We can discuss handling forks and badges after a standup if need be.